spec/react_on_rails/configuration_spec.rb

```diff
@@ -213,12 +213,12 @@ module ReactOnRails
       expect(ReactOnRails.configuration.auto_load_bundle).to eq(false)
     end
 
-    it "changes the configuration of the gem, such as setting the auto_load_bundle option to true" do
+    it "changes the configuration of the gem, such as setting the components_subdirectory option to 'something" do
```

Comment:
Why are we removing the test for `auto_load_bundle `? How about having both?

Reply:
Could leave test for both.